app/controllers/users.js

```diff
@@ -0,0 +1,24 @@
+const userService = require('../services/users');
+const bcrypt = require('bcrypt');
```

Comment:
For convention, in Wolox we import external libraries first separeted for a empty line before internal imports.